code/src/java/pcgen/cdom/base/CDOMObject.java

```diff
@@ -230,7 +230,7 @@ public final Formula remove(FormulaKey key)
 
 	public final Set<FormulaKey> getFormulaKeys()
 	{
-		return formulaChar == null ? Collections.<FormulaKey> emptySet()
+		return formulaChar == null ? Collections.emptySet()
```

Comment:
Why destroy the generics hint here?  This will introduce warnings back into the code...
I don't like this set of changes